libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/TitleBar.styles.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { IStyle, mergeStyleSets, IProcessedStyleSet } from "@fluentui/react";
+
+export interface ITitleBarStyles {
+  errorIcon: IStyle;
+  iconContainer: IStyle;
+  successIcon: IStyle;
+  titleBarLabel: IStyle;
+  titleBarNumber: IStyle;
+}
+
+export const titleBarStyles: () => IProcessedStyleSet<ITitleBarStyles> = () => {
+  return mergeStyleSets<ITitleBarStyles>({
+    errorIcon: {
+      color: "#d13438",
+      fontSize: "large",
+      fontWeight: "600"
+    },
+    iconContainer: {
+      position: "relative",
+      top: "2px"
+    },
+    successIcon: {
+      color: "#107c10",
```

Comment:
I might have missed, will you be using semantic colors in all places in coming PRs?

Reply:
Sorry, what are semantic colors? I just copied and pasted the values from the Fluent UI theme colors docs, but I can change it to use the get theme stuff too if that's preferred.

Reply:
yes you need to use the theme stuff, not specific values

Reply:
you can update in separate PR too I think, that's ok